pkg/kubelet/kubelet.go

```diff
@@ -1666,8 +1667,14 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		metrics.PodStartLatency.Observe(metrics.SinceInMicroseconds(firstSeenTime))
 	}
 
+	var statusAnnotations map[string]string
+	if apparmor.PodUsesAppArmor(pod) && apiPodStatus.Phase != api.PodFailed {
```

Comment:
I guess this won't cause any user visible concern since you didn't set something like

{apparmor.StatusAnnotationKey: apparmor.StatusDisabled} in status.